### ISSUE TITLE
error: [PIXI Hooks] !glTextures || !this.texturehook

### ISSUE THREAD

**Issue by reececomo** [Prozi/pixi-stats#4]
Hey 👋 awesome utility

Getting a console error `[PIXI Hooks] !glTextures || !this.texturehook` ([stats-gl.ts#L66](https://github.com/Prozi/pixi-stats/blob/3aeb746ed0ec995fb78958f59722aafc2870d6fb/src/stats-gl.ts#L66))

Not sure if it causes any larger issues - seems to be working OK anyway. Probably caused by a PixiJS version compatibility issue?

Details
- **pixi-stats version:** 1.3.6
- **PixiJS version:** 7.2.4

![Screenshot 2024-05-26 151221](https://github.com/Prozi/pixi-stats/assets/7390781/8510ed7c-0d00-48f6-b577-dd80b051dc3a)

**Comment by reececomo** [comment]
~Texture count is working, but draw calls (DC) is stuck on zero:~

<img src="https://github.com/Prozi/pixi-stats/assets/7390781/c6448dd2-3c9e-4c3a-96d0-5368ec640012" height=64px />

edit: Fixed DC - I was previously injecting a custom `renderer.gl.drawElements` to count draws, but removed that as pixi-stats does it instead 🙌

**Comment by nenjack** [comment]
@reececomo thanks for opening an issue I will check in upcoming weeks WTF XD

**Comment by nenjack** [comment]
@reececomo any update on this? sorry I forgot

**Comment by nenjack** [comment]
since I use this and it works I close until reopened

**Comment by reececomo** [comment]
Thanks @Prozi, I must have imagined I replied already - Close away 😄 

TL;DR: just an ominous console.error, no practical issue, just wanted to flag. Probably minor `pixi.js@7.2.4` compatibility issue 🤷‍♀️ (or I've got something funky in my config)
